Makefile

```
@@ -34,3 +36,7 @@ install-markdown-lint:
 markdown-lint:
 	@echo $(ALL_DOCS)
 	@for f in $(ALL_DOCS); do echo $$f; $(MARKDOWN_LINT) -c .markdownlint.yaml $$f || exit 1;	done
+
+.PHONY: table-generation
+table-generation:
+	docker run --rm -v $(PWD)/semantic_conventions:/source -v $(PWD)/specification:/spec thisthatdc/semconvgen -f /source markdown -md /spec
```

```suggestion
	docker run --rm -v $(PWD)/semantic_conventions:/source -v $(PWD)/specification:/spec otel/semconvgen -f /source markdown -md /spec
```
Should be working now that https://github.com/open-telemetry/build-tools/pull/5 was merged and the image published to https://hub.docker.com/r/otel/semconvgen.